packaging_automation/tests/test_update_package_properties.py

```diff
@@ -91,35 +69,20 @@ def test_prepend_latest_changelog_into_debian_changelog():
         os.remove(changelog_file_path)
 
 
-def test_prepend_latest_changelog_into_debian_changelog_10_0_3_already_included():
-    refer_file_path = f"{TEST_BASE_PATH}/files/debian.changelog_include_10_0_3.refer"
-    changelog_file_path = f"{TEST_BASE_PATH}/files/debian.changelog"
-    copyfile(refer_file_path, changelog_file_path)
-    latest_changelog = changelog_for_tag(GITHUB_TOKEN, PROJECT_NAME, TAG_NAME)
-    changelog_param = default_changelog_param_for_test(latest_changelog, datetime.now())
-    try:
-
-        with pytest.raises(ValueError):
-            prepend_latest_changelog_into_debian_changelog(changelog_param, changelog_file_path)
-    finally:
-        os.remove(changelog_file_path)
-
-
 def verify_prepend_debian_changelog(changelog_file_path):
     with open(changelog_file_path, "r", encoding=DEFAULT_ENCODING_FOR_FILE_HANDLING,
               errors=DEFAULT_UNICODE_ERROR_HANDLER) as reader:
         content = reader.read()
-        latest_changelog = get_last_changelog_content_from_debian(content)
-    with open(f"{TEST_BASE_PATH}/files/verify/expected_debian_latest_v10.0.3.txt", "r",
+    with open(f"{TEST_BASE_PATH}/files/verify/debian_changelog_with_10.0.3.txt", "r",
               encoding=DEFAULT_ENCODING_FOR_FILE_HANDLING,
               errors=DEFAULT_UNICODE_ERROR_HANDLER) as reader:
         expected_content = reader.read()
-    are_strings_equal(expected_content, latest_changelog)
+    assert content == expected_content
 
 
 def test_convert_citus_changelog_into_rpm_changelog():
```

Comment:
The function name here no longer make sense. We do not convert anything but create a changelog from scratch.

Reply:
Fixed